config/config_DEPRECATED.ts

```diff
@@ -408,12 +408,10 @@ export function getAccountId(nameOrId?: string | number): number | undefined {
   function setNameOrAccountFromSuppliedValue(
     suppliedValue: string | number
   ): void {
-    if (typeof suppliedValue === 'number') {
-      accountId = suppliedValue;
-    } else if (/^\d+$/.test(suppliedValue)) {
-      accountId = parseInt(suppliedValue, 10);
-    } else {
+    if (typeof suppliedValue === 'string') {
       name = suppliedValue;
```

Comment:
I think in some cases, our `--account` flag supports either a string (`my-prod-account`) or a number that represents the account Id (`1248533`).  I think in both cases it will show up in this util as a string. I'm wondering with these changes would I still be able to do something like `hs account use 123456` where `123456` is my account id?

For example, here's the output of `hs account use --help`
<img width="781" alt="image" src="https://github.com/user-attachments/assets/ce54aba8-ddd8-4290-938f-863884e5edc2" />